src/contrib/inference/AdvancedSMCExtensions.jl

```diff
@@ -65,19 +65,6 @@ function Sampler(alg::PMMH, model::Model, s::Selector)
         space = union(space, sub_alg.space)
     end
 
-    # Sanity check for space
```

Comment:
Why removing sanity checks? Do we have an alternative now?

Reply:
Because `get_pvars` is not defined in the type of the model now; the variable names can be passed in by the user using `NamedDist`.

Reply:
Previously, the parameter symbols were part of the type of `model`, so this was simple. Now, we would need to call `VarInfo(model)` and get the symbols of the result to know the symbols of the variables. I can try saving the symbols in the `Model` struct at construction by doing something similar to `VarInfo(model)` but because the user is free to change the variable names of `model` after construction, this approach is not good.

Reply:
I see. It is useful to have some sanity checking for variable assignment in case some variables are not assigned. But I agree that we can address this issue in a separate effort. Can you create a new issue for this issue?

Reply:
Sure

Reply:
https://github.com/TuringLang/Turing.jl/issues/1012